src/test/kotlin/uk/gov/dluhc/printapi/testsupport/testdata/model/SourceApplicationRemovedMessageBuilder.kt

```diff
@@ -0,0 +1,16 @@
+package uk.gov.dluhc.printapi.testsupport.testdata.model
+
+import uk.gov.dluhc.printapi.messaging.models.ApplicationRemovedMessage
+import uk.gov.dluhc.printapi.messaging.models.SourceType
+import uk.gov.dluhc.printapi.testsupport.testdata.aValidSourceReference
+import uk.gov.dluhc.printapi.testsupport.testdata.getRandomGssCode
+
+fun buildApplicationRemovedMessage(
```

Comment:
Could rename the class name to be ApplicationRemovedMessageBuilder as we took `Source` out of it's name